test/test_unary_ufuncs.py

```diff
@@ -1020,6 +1020,17 @@ def test_hardsigmoid_backward(self, device, dtype):
         out.backward(torch.ones_like(inputTensor))
         self.assertEqual(inputTensor.grad, expetedTensor)
 
+    @dtypesIfCUDA(torch.float, torch.double, torch.bfloat16)
```

Comment:
From the test failure, you need to remove `bfloat16` from here as it is not supported.